### ISSUE TITLE
Does not run on Clear Linux OS

### ISSUE THREAD

**Issue by probonopd** [AlbrechtL/welle.io#319]
Does not run on Clear Linux OS:

```
      1175:	/usr/lib64/libfontconfig.so.1: error: symbol lookup error: undefined symbol: FT_Done_MM_Var (fatal)
This application failed to start because it could not find or load the Qt platform plugin "xcb"
in "".

Available platform plugins are: xcb.

Reinstalling the application may fix this problem.
Aborted (core dumped)
```

Not bundling FreeType solves this.

**Comment by AlbrechtL** [comment]
I don't really understand the problem.
> Not bundling FreeType solves this.

What do you mean with this?

**Comment by AlbrechtL** [comment]
Thanks for reporting this issue. Can you give me a hint what I can do?

I mean you created the initial [.travis.yml](https://github.com/AlbrechtL/welle.io/blob/571532eae289cba8927b8c0442763bc4fecc4a37/.travis.yml).

**Comment by probonopd** [comment]
Probably just running a new build would be sufficient, since the AppImage tools have the FreeType library blacklisted nowadays.

**Comment by AlbrechtL** [comment]
When did you blacklist the FreeType library?

Can you test the latest nightly build? https://bintray.com/albrechtl/welle.io/welle.io_nightly#files

**Comment by probonopd** [comment]
> When did you blacklist the FreeType library?

8 months ago, https://github.com/AppImage/AppImages/commit/9082bbe3b924b656d8675da64a69c6b2b32ac7b4

**Comment by AlbrechtL** [comment]
The device should work anyway. The error message is a bug.

**Comment by probonopd** [comment]
Indeed! When launching the AppImage linked above as root, I do get a read error, but nevertheless I can search for stations and listen to them. So, working for me (more or less :-))

![works](https://user-images.githubusercontent.com/2480569/48317961-427a7200-e5f1-11e8-9db4-5e02aef39f36.png)
